services/openshift/templates/eap64-mysql-persistent-s2i.json

```diff
@@ -446,7 +446,7 @@
                         }
                     },
                     "spec": {
-                        "serviceAccount": "eap-service-account",
+                        "serviceAccountName": "eap-service-account",
```

Comment:
Looks like it changed https://github.com/jboss-openshift/application-templates/commit/373e1de465c58018c129f85229be83f7d047499b#diff-cbe42c8c682826a475a22ba56a93ae81 so @LalatenduMohanty just  updated it (But we have to make sure that is also work with ose-3.1.1.6 which we are going to ship with CDK).